### ISSUE TITLE
Failing Check - F821 undefined name 'cinfo' - helpers.py

### ISSUE THREAD

**Issue by MattIBM** [MattIBM/discord-wilfred#18]

**Comment by MM-coder** [comment]
Note, reffers to the following:

```py
def insert_db_user(member):
    try:
        sql.execute_query("ibm.db", "INSERT INTO Members (UserID) VALUES ('%s')" % (member.id))
    except:
        log.warn("User already exists in Database")
        try:
            cinfo(member.name)
        except:
            pass```

**Comment by MM-coder** [comment]
Note 76da6bd

**Comment by MattIBM** [comment]
@MM-coder You don't need to note everything in issues

**Comment by MattIBM** [comment]
Fixed in #19
